test/vstest.console.UnitTests/Processors/RunSpecificTestsArgumentProcessorTests.cs

```diff
@@ -101,6 +101,51 @@ public void CapabilitiesShouldReturnAppropriateProperties()
 
         #region RunSpecificTestsArgumentExecutorTests
 
+        [TestMethod]
+        public void InitializeShouldThrowIfArguemntIsNull()
+        {
+            CommandLineOptions.Instance.Reset();
+            var testRequestManager = new TestRequestManager(CommandLineOptions.Instance, new TestPlatform(), TestLoggerManager.Instance, TestRunResultAggregator.Instance, this.mockTestPlatformEventSource.Object, this.inferHelper, this.mockMetricsPublisherTask);
+            var executor = GetExecutor(testRequestManager);
+            Assert.ThrowsException<CommandLineException>(() => { executor.Initialize(null); });
+        }
+
+        [TestMethod]
+        public void InitializeShouldThrowIfArgumentIsEmpty()
+        {
+            CommandLineOptions.Instance.Reset();
+            var testRequestManager = new TestRequestManager(CommandLineOptions.Instance, new TestPlatform(), TestLoggerManager.Instance, TestRunResultAggregator.Instance, this.mockTestPlatformEventSource.Object, this.inferHelper, this.mockMetricsPublisherTask);
+            var executor = GetExecutor(testRequestManager);
```

Comment:
nit: single blank line between `arrange`, `act` and `assert` == awesome readability :)

Reply:
noted :)